src/vs/workbench/contrib/positronConnections/browser/components/schemaNavigation.css

```diff
@@ -0,0 +1,112 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (C) 2024 Posit Software, PBC. All rights reserved.
+ *  Licensed under the Elastic License 2.0. See LICENSE.txt for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.connections-items-container {
+	height: 100%;
+	position: relative;
+	left: 0.5px;
+}
+
+.connections-item {
+	display: flex;
+	cursor: pointer;
+	align-items: center;
+	height: 26px;
+}
+
+.connections-details {
+	flex-grow: 1;
+	overflow: hidden;
+	text-overflow: ellipsis;
+	white-space: nowrap;
+	font-family: monospace;
+	display: flex;
+	align-items: center;
+}
+
+.connections-dtype {
+	color: var(--vscode-positronSideActionBar-disabledForeground);
+	font-weight: 300;
+}
+
+.connections-dtype::before {
+	content: ":";
+	color: var(--vscode-positronVariables-foreground);
+	margin-left: 3px;
+	margin-right: 3px;
```

Comment:
NIT: `margin: 0 3px;`

Reply:
Resolved in b4fa6f9c2c2a306962b074a0e497fab44026f082